traja/dataset/pituitary_gland.py

```diff
@@ -0,0 +1,151 @@
+import numpy as np
+import pandas as pd
+from numpy import exp
+from numba import jit
+from scipy.integrate import odeint
+from pyDOE2 import lhs
+
+
+# PyTest will not compute coverage correctly for @jit-compiled code.
+# Thus we must explicitly suppress the coverage check.
+@jit
+def pituitary_ode(w, t, p):  # pragma: no cover
```

Comment:
maybe add type hints for return values `-> Tuple[float, float, float, float]` with `from typing import Tuple` at the top